src/dftbp/reks/reksen.F90

```diff
@@ -821,8 +1039,54 @@ subroutine getPseudoFock_(Fc, Fa, orbFON, Nc, Na, fock)
     call adjointLowerTriangle(fock)
 
   end subroutine getPseudoFock_
+#:endif
 
 
+#:if WITH_SCALAPACK
+  !> Avoid changing the order of MOs
+  !> Required number of cycles increases as the number of shift increases
```

Comment:
Again, see https://dftbplus-develguide.readthedocs.io/en/latest/fortranstyle.html#comments (double-bang for the second and subsequent lines).